ynput/ayon-frontend
# Enhancement: saving editor changes

Based on feedback from @LiborBatek 🚀

### Description

- You can now instantly save a single field value by hitting ctrl/cmd enter inside the field. Only that field will save, other unsaved changes will remain.
- Changed the styles of changes in editor
- fixed some dialogs, clicking outside now closes them.
- Fixed z-index issue with date picker

![image](https://github.com/ynput/ayon-frontend/assets/49156310/7c65db70-1d35-4242-98b2-487d9cae856d)

### Testing

1. Open editor and make a load of unsaved changes and make some new folders/tasks.
2. Change one specific field and then hit ctrl/cmd (windows/mac) enter to saved just that one field.
3. The field should save (removed changes styles) and a success toast appears
4. All other changes should stay the same.
